code/data_structures/Linked_lists/Header_Linked_List.cpp

```diff
@@ -0,0 +1,80 @@
+//Part of Cosmos by OpenGenus Foundation
+//Contributed by Tushar Gautam (Tushark21)
+
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main(){
+
+struct node{
+	int info;
+	node *next;
+}*ptr,*head,*start;
+
+int c=1,k=0,ele,data;
+
+ptr=new node;
+ptr->next=NULL;
+start=head=ptr;
+
+while(c<4 && c>0){
+	cout<<"1.Insert\n2.Search Item\n3.Link List\n";
+	cin>>c;
+
+	switch(c){
```

Comment:
Kindly use functions instead of switch case as this will make the code modular and adaptable. Make the changes to get this merged 👍